app/modules/transcribe/aggregations/aggregations.factory.js

```diff
@@ -40,9 +40,13 @@ function AggregationsFactory($q, SubjectsFactory, zooAPI, zooAPIConfig, zooAPIPr
             .then(_formatAggregations)
             .then(function (newAggregations) {
                 _aggregations = _aggregations.concat(newAggregations);
-            })
+            }, loadAggregationsError);
```

Comment:
You can use a `catch` block instead of the second argument to `Then`, which might make it clearer what this is doing.

Reply:
Ah yes, sorry I missed that. I used `catch` in the other functions though.

Reply:
Yes, you've used `catch` inside `catch`, which is odd.